--- ../../anagramarama/src/linked.c	2010-06-14 20:51:00.000000000 -0400
+++ linked.c	2010-06-15 17:03:19.000000000 -0400
@@ -155,7 +155,7 @@
 
 	if (!duplicate){
 	        len = strlen(anagram+1);
-	        newNode->anagram = malloc(sizeof(char)*len);
+	        newNode->anagram = malloc(sizeof(char)*(len + 1));
 	        strcpy(newNode->anagram, anagram);
 	        newNode->length = len + 1;
 	        newNode->found = 0;
